torch/_inductor/codegen/wrapper.py

```diff
@@ -1333,7 +1334,7 @@ def codegen_input_size_and_nan_asserts(self) -> None:
     # that stream caching happens per graph instance. this
     # is important for nested subgraph codegening.
     def write_get_raw_stream(self, device_idx: int, graph_name: str) -> str:
-        self.write_get_raw_stream_header_once()
+        self.write_get_raw_stream_header()
```

Comment:
standalone_compile may need additional imports of `get_raw_stream`

Reply:
Why additional imports are needed? Can those be added as cache key if repeated imports are needed

Reply:
> Currently, write_get_raw_stream_header_once relies on cache_on_self to import once. When generating subgraph, it calls SubgraphPythonWrapperCodegen.write_get_raw_stream_header_once() (cache_on_self) which calls PythonWrapperCodegen.write_get_raw_stream_header_once() (cache_on_self).

Is the import for the subgraph wrapper not added to the global scope?

Reply:
No. A new wrapper code is generated for each subgraph.
https://github.com/pytorch/pytorch/blob/8590c3a66b4adb77b59645dd12bf44695ac3b994/torch/_inductor/scheduler.py#L5166-L5173

Reply:
yes for the output_code. But no for `config.triton.autotune_at_compile_time`, which creates a separate code to run IIUC.